controller/token.go

```diff
@@ -30,6 +30,11 @@ import (
 	"golang.org/x/oauth2"
 )
 
+const (
+	DevUsername = "developer"
```

Comment:
Could we make these internal to the package?

Reply:
actually, it's also used in the test, which is not in the same pkg

Reply:
no we can't as we used it in test like @xcoulon  mentioned

Reply:
You could convert your test to white_box then.

Reply:
Oh.. it's actually just a string. Then you really should have just use the same string in your tests instead of exporting internal constants. But it's not that critical.

Reply:
yes, that could work, too.